app/src/main/java/com/github/swent/echo/data/model/Event.kt

```diff
@@ -9,10 +9,14 @@ data class Event(
     val eventId: String,
     val creatorId: String,
     val organizerId: String,
+    val organizerName: String,
```

Comment:
we might prefer to have the objects directly available here rather than the ids and name(s)... (so the organizing association and the creating user(Profile)...